src/components/PeoplePage.tsx

```diff
@@ -1,8 +1,105 @@
+/* eslint-disable max-len */
+import { useParams, useSearchParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { getParent } from '../utils/searchHelper';
 
 export const PeoplePage = () => {
+  const [persons, setPersons] = useState<Person[]>([]);
+  const [loader, setLoader] = useState(false);
+  const { personSlug } = useParams();
+
+  const [searchParams] = useSearchParams();
+
+  const currentCenturies = searchParams.getAll('centuries') || [];
+  const currentQuery = searchParams.get('query') || '';
+  const currentSex = searchParams.get('sex') || '';
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  useEffect(() => {
+    setLoader(true);
+
+    setTimeout(() => {
+      getPeople()
+        .then((data) => {
+          setPersons(data);
+          setLoader(false);
+        })
+        .catch(() => {
+          setLoader(false);
```

Comment:
Consider moving setLoader(false) to a `.finally` block to not repeat yourself

Read more [here](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally)